reviews/2022-cdk8s-annual.md

```diff
@@ -0,0 +1,101 @@
+## Background
+
+CDK For Kubernetes (cdk8s) is an open-source software development framework for defining Kubernetes 
+applications and reusable abstractions. Users leverage familiar programming languages and rich object-oriented APIs to 
+create standard Kubernetes manifests which can be applied to any Kubernetes cluster.
+
+CDK8s joined the CNCF on November 10, 2022.
+
+## Annual Review
+
+The following contains the requested content for the annual review, as described [here](https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md).
+
+### Include a link to your project’s devstats page.
+
+The full devstats page for cdk8s is available here: https://cdk8s.devstats.cncf.io/d/8/dashboards
+
+* Maintaining a steady rate of merged PRs across our repositories. 
+* There are 7 [community contributions](https://cdk8s.devstats.cncf.io/d/56/company-commits-table?orgId=1&from=now-1y&to=now&var-repogroups=All&var-companies=Palo%20Alto%20Networks&var-companies=Cisco&var-companies=GridLine) 
+from 3 different users, in 3 different organizations (Palo Alto Networks, GridLine, Cisco). There is an additional [contribution](https://github.com/cdk8s-team/cdk8s-cli/pull/324) 
+that doesn’t show up on devstats, probably because the user isn’t associated with any company.
+
+### How many maintainers do you have, and which organizations are they from?
+
+The project currently has 4 maintainers, all from AWS:
+
+* [@iliapolo](https://github.com/iliapolo)
+* [@vinayak-kukreja](https://github.com/vinayak-kukreja)
+* [@sumupitchayan](https://github.com/sumupitchayan)
+* [@ryparker](https://github.com/ryparker)
+
```

Comment:
have we identified any potential maintainers outside of AWS?

Reply:
Yes but do you think we should mention them? we haven't spoken to them about it yet.

Reply:
I'm not going to wait on this as its still to ambiguous